go.mod

```diff
@@ -3,6 +3,8 @@ module sigs.k8s.io/e2e-framework
 go 1.16
 
 require (
+	k8s.io/api v0.21.0
 	k8s.io/apimachinery v0.21.0
 	k8s.io/client-go v0.21.0
+	sigs.k8s.io/controller-runtime v0.8.3
```

Comment:
Now that we're relying on ctrl-runtime, we should make sure we're using latest stable. It looks like it's v0.9.0.